manim/mobject/value_tracker.py

```diff
@@ -87,24 +118,84 @@ def interpolate(self, mobject1, mobject2, alpha, path_func=straight_path):
 
 
 class ExponentialValueTracker(ValueTracker):
-    """
-    Operates just like ValueTracker, except it encodes the value as the
+    """Operates just like ValueTracker, except it encodes the value as the
     exponential of a position coordinate, which changes how interpolation
-    behaves
+    behaves.
+
+    Note that ExponentialValueTracker does not handle non-positive values.
+
+    Examples
+    --------
+    .. manim:: ExponentialValueTrackerExample
+
+        class ExponentialValueTrackerExample(Scene):
+            def construct(self):
+                number_line = NumberLine()
+                pointer = Vector(DOWN)
+                label = MathTex("x").add_updater(lambda m: m.next_to(pointer, UP))
+
+                pointer_value = ExponentialValueTracker(4)
+                pointer.add_updater(
+                    lambda m: m.next_to(
+                                pointer_value.get_value() * RIGHT,
+                                UP
+                            )
+                )
+                self.add(number_line, pointer,label)
+
+                self.play(pointer_value.animate.set_value(0.5))
+                self.wait(0.5)
+                self.play(pointer_value.animate.set_value(6))
+                self.wait(0.5)
+                self.play(pointer_value.animate.set_value(3))
+                self.wait(0.5)
+                self.play(pointer_value.animate.set_value(2))
+                self.wait(0.5)
     """
 
     def get_value(self):
+        """Get the current value of the ExponentialValueTracker."""
         return np.exp(ValueTracker.get_value(self))
 
     def set_value(self, value):
+        """Set a new scalar value to the ExponentialValueTracker. The value cannot
+        be non-positive."""
         return ValueTracker.set_value(self, np.log(value))
 
 
 class ComplexValueTracker(ValueTracker):
+    """Operates like ValueTracker, except it encodes a complex-valued
+    parameter as opposed to a real-valued parameter.
+
+    Examples
+    --------
+    .. manim:: ComplexValueTrackerExample
+
+        class ComplexValueTrackerExample(Scene):
+            def construct(self):
+                tracker = ComplexValueTracker(-2+1j)
+                dot = Dot().add_updater(
+                    lambda x: x.move_to(tracker.points)
+                )
+
+                self.add(NumberPlane(), dot)
+
+                self.play(tracker.animate.set_value(3+2j))
+                self.play(tracker.animate.set_value(tracker.get_value() * 1j))
+                self.play(tracker.animate.set_value(tracker.get_value() - 2j))
+                self.play(tracker.animate.set_value(tracker.get_value() / (-2 + 3j)))
+    """
+
     def get_value(self):
+        """Get the current value of the ComplexValueTracker. This value changes
+        continuously when :attr:`animate` for the ComplexValueTracker is called."""
         return complex(*self.points[0, :2])
 
     def set_value(self, z):
```

Comment:
Same comment regarding the note on `.animate`. It might make sense to point out that complex values are actually stored as point coordinates (i.e., a + jb is stored internally as [a, b, 0]).

Reply:
I don't think it makes sense to provide those implementation details to the user. It's useful from a development standpoint, but a user shouldn't need to know that it's dependent on an internal array.

Reply:
After thinking about it, I think it does make sense to distinctly mention the points array, because it can be useful as shorthand for using the complex value directly. Added documentation to the get_value method.